PROTO_tests/tests/symbol_table_test.py

```diff
@@ -721,38 +721,38 @@ def test_attach_weak_binding(self, dtype):
             str(a)
 
     def test_symentry_cleanup(self):
-        pda = self.make_pdarray(ak.int64, 10)
+        pda = self.make_pdarray(ak.int64, 10)  # noqa: F841
```

Comment:
similarly, i would just add this to `per-file-ignores` in `setup.cfg`

Reply:
Updated